Add base fee to glossary (#5137)
ethereum/ethereum-org-website: src/content/glossary/index.md

## B {#section-b}

### Base Fee {#base-fee}

> Every Block has a reserve price called the "base fee". It is the minimum fee a user pays to include a transaction in the next block. The base fee helps in making the transaction more predictable.

Reviewer: - Block shouldn't be capitalised mid-sentence
- We should link to block and gas definitions in the glossary
- I think we should drop the last sentence, we should only define base fee in the glossary, not explain why it was introduced.

```suggestion
Every [block](#block) has a reserve price known as the 'base fee'. It is the minimum [gas](#gas) fee a user must pay to include a transaction in the next block.
```

<DocLink to="developers/docs/gas/">

>   Gas & Fees

Reviewer: ```suggestion
  Gas and fees
```
